### ISSUE TITLE
`PointCloud` class ignores normals 

### ISSUE THREAD

**Issue by Sentient07** [mikedh/trimesh#1780]
Hi,

I'm loading a point cloud with normals from an xyz file. `pointcloud.vertices` contains only the vertex positions and the class [PointCloud](https://github.com/mikedh/trimesh/blob/main/trimesh/points.py#L401) doesn't seem to provide access to normals. I think it would be good to have a class variable `normals` so that one can access normals like in the case of `mesh.vertex_normals`.

Thanks

**Comment by whateverforever** [comment]
Good point, it also doesn't support `.scale`, and likely other stuff. I'm currently ball pivoting my pointclouds so I can use it with trimesh. Not sure if the PointCloud class has a real benefit existing at all, after all, is a point cloud not just a mesh without connectivity?
